queries.sql

```diff
@@ -23,3 +23,30 @@ SELECT * FROM animals WHERE name NOT IN ("Gabumon");
 
 -- 8.0 Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+-- Part 2 Questions
+
+-- 1. How many animals are there?
+
+SELECT COUNT(*) FROM animals;
+
+-- 2. How many animals have never tried to escape?
+SELECT COUNT(*) FROM animals WHERE escape_attempts=0;
+
```

Comment:
- [ ] It would be better if we `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set. And also Commit the transaction